packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -47,6 +47,7 @@ type FieldProps = {
   maxLength?: number;
   minLength?: number;
   autoFocus?: boolean;
+  options?: object;
```

Comment:
I don't think this needs to be added here - options isn't used within the Field component